pyscriptjs/tests/integration/support.py

```diff
@@ -290,7 +329,8 @@ class ConsoleMessageCollection:
 
     Usage:
 
-      console.log.messages: list of ConsoleMessage with type=='log'
+      console.log.messages: list of ConsoleMessage with type==
+    'log'
```

Comment:
newline seems unnecessary here unless some linter ruined this?

Reply:
good catch, it was introduced by mistake